sdk/v1/client.go

```diff
@@ -141,6 +141,35 @@ func (cli *Client) PatchJob(ctx context.Context, headers client.Headers, jobID s
 	return respHeaders, nil
 }
 
+// GetTask Get a specific task for a given reindex job
+func (cli *Client) GetTask(ctx context.Context, reqheader client.Headers, jobID, taskName string) (client.RespHeaders, *models.Task, error) {
```

Comment:
Again worth making client.RespHeaders a pointer for the same reason as models.Task